### PR TITLE
Bump data-model -> 0.20.16

### PR DESCRIPTION
**Pull request by brinxmat**

### CODE REVIEW

**Review by truhacevkir — APPROVED** [review]

File: cristin-import/src/main/java/no/unit/nva/cristin/patcher/CristinPatcher.java [hunk 1]
```diff
@@ -27,8 +27,8 @@ public static ParentAndChild updateChildPublication(ParentAndChild parentAndChil
         var parentPublicationInstance =
             parentAndChild.getParentPublication().getEntityDescription().getReference().getPublicationInstance();
         if (parentChildPublicationContextMatch(childContext, parentPublicationInstance)) {
-            var chapter = (Chapter) childContext;
-            chapter.setPartOf(createPartOfUri(parentAndChild.getParentPublication()));
+            var chapter = (Anthology) childContext;
```
Comment (ketilaa) [inline cristin-import/src/main/java/no/unit/nva/cristin/patcher/CristinPatcher.java:30]: adjust variable name to anthology?

File: scopus-import/src/main/java/no/sikt/nva/scopus/conversion/PublicationContextCreator.java [hunk 2]
```diff
@@ -111,9 +111,9 @@ public Book createReport() {
             () -> new Report(bookSeries, seriesTitle, seriesNumber, publishingHouse, isbnList)).orElseThrow();
     }
 
-    public Chapter createChapter() {
+    public Anthology createChapter() {
```
Comment (ketilaa) [inline scopus-import/src/main/java/no/sikt/nva/scopus/conversion/PublicationContextCreator.java:114]: Method name?

File: scopus-import/src/test/java/no/sikt/nva/scopus/ScopusHandlerTest.java [hunk 3]
```diff
@@ -468,8 +468,8 @@ void shouldReturnPublicationContextChapterWhenScopusXmlHasCitationTypeChEvenIfSr
         var s3Event = createNewScopusPublicationEvent();
         var publication = scopusHandler.handleRequest(s3Event, CONTEXT);
         var actualPublicationContext = publication.getEntityDescription().getReference().getPublicationContext();
-        assertThat(actualPublicationContext, instanceOf(Chapter.class));
-        var actualPartOfUri = ((Chapter) actualPublicationContext).getPartOf();
+        assertThat(actualPublicationContext, instanceOf(Anthology.class));
```
Comment (ketilaa) [inline scopus-import/src/test/java/no/sikt/nva/scopus/ScopusHandlerTest.java:471]: still traces of `partOf` here

File: scopus-import/src/test/java/no/sikt/nva/scopus/ScopusHandlerTest.java [hunk 4]
```diff
@@ -541,8 +541,8 @@ void shouldReturnPublicationContextChapterWhenSrcTypeIsPAndIsbnExists() throws I
         var s3Event = createS3Event(uri);
         var publication = scopusHandler.handleRequest(s3Event, CONTEXT);
         var actualPublicationContext = publication.getEntityDescription().getReference().getPublicationContext();
-        assertThat(actualPublicationContext, instanceOf(Chapter.class));
-        var actualPartOfUri = ((Chapter) actualPublicationContext).getPartOf();
+        assertThat(actualPublicationContext, instanceOf(Anthology.class));
```
Comment (ketilaa) [inline scopus-import/src/test/java/no/sikt/nva/scopus/ScopusHandlerTest.java:544]: still `partOf`

**Review by ketilaa — APPROVED** [review]
Approved with some comments on variable and method names.

**Pull request merged**